<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GBSBizDeviceInfo</class>
 <widget class="QWidget" name="GBSBizDeviceInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1238</width>
    <height>920</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="wdgLocalInfo" native="true">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>350</width>
     <height>359</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="wdgRemoteInfo" native="true">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>40</y>
     <width>350</width>
     <height>359</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="wdgRunningInfo" native="true">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>40</y>
     <width>350</width>
     <height>359</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="wdgStreamInfo" native="true">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>420</y>
     <width>1090</width>
     <height>468</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
